guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -1,95 +1,133 @@
 [id="configuring-repositories_{context}"]
 = Configuring repositories
 
-Use these procedures to enable the repositories required to install {ProductName}.
+ifdef::orcharhino[]
+Ensure the repositories required to install {ProductName} are enabled on your {EL} host.
```

Comment:
How is this not a prerequisite?

Reply:
This is intentional for `orcharhino`. There is no procedure for it as of now.

Reply:
installing orcharhino Server and orcharhino Proxies differs to upstream and Satellite. So I am OK if this just says "provide content" fow now.